fix name of FUSE library to search for (libfuse3.so for FUSE 3.x)
author: Kenneth Hoste (HPC-UGent)
--- fusepy-2.0.2/fuse.py.orig	2018-10-26 08:56:44.092750000 +0200
+++ fusepy-2.0.2/fuse.py	2018-10-26 08:56:57.559706777 +0200
@@ -63,7 +63,7 @@
     _libfuse_path = (find_library('fuse4x') or find_library('osxfuse') or
                      find_library('fuse'))
 else:
-    _libfuse_path = find_library('fuse')
+    _libfuse_path = find_library('fuse3')
 
 if not _libfuse_path:
     raise EnvironmentError('Unable to find libfuse')
